ui/pages/Project/components/RnaSeqResultPage.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Loader, Grid, Dropdown } from 'semantic-ui-react'
+
+import { getGenesById, getIndividualsByGuid, getRnaSeqDataByIndividual, getRnaSeqSignificantJunctionData } from 'redux/selectors'
+import { RNASEQ_JUNCTION_PADDING, TISSUE_DISPLAY } from 'shared/utils/constants'
+import DataLoader from 'shared/components/DataLoader'
+import FamilyReads from 'shared/components/panel/family/FamilyReads'
+import RnaSeqJunctionOutliersTable from 'shared/components/table/RnaSeqJunctionOutliersTable'
+import { loadRnaSeqData } from '../reducers'
+import { getRnaSeqDataLoading } from '../selectors'
+
+const RnaSeqOutliers = React.lazy(() => import('./RnaSeqOutliers'))
+
+const OUTLIER_VOLCANO_PLOT_CONFIGS = [
+  {
+    key: 'outliers',
+    getLocation: (({ geneId }) => geneId),
+    searchType: 'genes',
+    title: 'Expression Outliers',
+    formatData: data => data,
+  },
+  {
+    key: 'spliceOutliers',
+    getLocation: (({ chrom, start, end }) => `${chrom}:${Math.max(1, start - RNASEQ_JUNCTION_PADDING)}-${end + RNASEQ_JUNCTION_PADDING}`),
+    searchType: 'regions',
+    title: 'Splice Junction Outliers',
+    formatData: (data, tissueType) => data.filter(outlier => outlier.tissueType === tissueType),
+  },
+]
+
+const OutliersTableWithReads = props => <FamilyReads layout={RnaSeqJunctionOutliersTable} noTriggerButton {...props} />
```

Comment:
it will be more readable not to have this be its own class, and just directly render `FamilyReads` below with the props you want